projects/ccf-rui/src/app/modules/header/header.component.scss

```diff
@@ -12,7 +12,8 @@
     display: flex;
     align-items: center;
     font-weight: lighter;
-    font-size: 2rem;
+    font-size: 1.5rem;
+    color: #8289A3;
```

Comment:
Colors go into the material theme files. If @axdanbol is around he can show you how it works, otherwise we'll show you tomorrow.

Reply:
Oh I see, shoot!